admin/src/main/resources/io/buoyant/admin/js/src/router_client.js

```diff
@@ -99,10 +103,37 @@ define([
       return summary;
     }
 
-    return function (metricsCollector, routers, client, $metricsEl, routerName, $chartEl, colors, $toggleLinks, shouldExpandInitially) {
+    function barChartColorFn(percent) {
+      return percent < 0.5 ? "orange" : "green";
+    }
+
+    function barPercentCalc(data) {
```

Comment:
I'd like to see these barchart-related methods in their own function or object named `lbBarChart` to help separate them from the rest of the router client. Currently it's hard to tell whether or not these methods rely on the router client's scope. I'd start by defining them outside of `RouterClient`.

Same situation for `RouterSummary` and `retriesBarChart`